### PR TITLE
adding possibilty of multiple buttons in hash and its render; fixes #22

### LINKED ISSUE #22: Provide multiple website links in Popup
We need the possibility to show multiple links in the country popup.

The JSON structure will be as follows:

```
{
  'AT': {
    'name': 'Austria',
    'websites': [
      { 'label': 'something', url: 'http://www.yfu.at' },
      { 'label': 'another thing', url: 'http://www.yfu-austria.at' }
    ]
  }
}
```

### PR DESCRIPTION
**Pull request by brianstanley**

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -47,13 +47,13 @@
 <body>
 <div id="world-map" data-colors = '{"hover": "red", "highlighted":  "blue"   }'  data-countries=' {
 "AT":
-{"name": "Austria", "main_image": "src", "description": "brief description"}
+{"name": "Austria", "main_image": "src", "websites": "brief description"}
```
Comment (icatcher-at) [inline demo/index.html:50]: What is `main_image` exactly?

File: demo/index.html [hunk 2]
```diff
@@ -47,13 +47,13 @@
...
 ,
 "AR":
-{"name": "Argentina", "main_image": "http://estaticos2.catai.es/content/media/fotos/samples/large/paisaje-austriaco.jpg", "description": "The best country in the world"}
+{"name": "Argentina", "main_image": "http://estaticos2.catai.es/content/media/fotos/samples/large/paisaje-austriaco.jpg", "websites": "The best country in the world"}
 ,
 "BR":
-{"name": "Brasil", "country_flag_image": "src", "main_image": "http://www.bvmemorial.fapesp.br/var/home/memorial/htdocs//local/Image/ban_brasil.JPG", "description": "Brasil is worst in football than Argentina"}
+{"name": "Brasil", "country_flag_image": "src", "main_image": "http://www.bvmemorial.fapesp.br/var/home/memorial/htdocs//local/Image/ban_brasil.JPG", "websites": "Brasil is worst in football than Argentina"}
```
Comment (icatcher-at) [inline demo/index.html:56]: Can you explain `country_flag_image` as well, to me?

File: demo/index.html (hunk 1 above)
Comment (brianstanley) [inline demo/index.html:50]: shit I forget to delete D: old data is not using anymore in the example ... deleting...

File: demo/index.html (hunk 2 above)
Comment (brianstanley) [inline demo/index.html:56]: auch nicht

**Pull request merged**